assets/component-product-grid.css

```diff
@@ -1,21 +1,21 @@
 .product-grid .grid__item {
-  padding-bottom: 2rem;
+  padding-bottom: calc(2rem + var(--page-width-margin));
 }
 
 .product-grid.negative-margin {
-  margin-bottom: -2rem;
+  margin-bottom: calc(-2rem - var(--page-width-margin));
```

Comment:
Here I wonder if it should impact the values on mobile, as nothing really changes on mobile compared to the `1200px` version.

Reply:
but maybe for overall consistency. So it's not tight on mobile vs airy on desktop.

Reply:
I think it does make sense to keep it on mobile as I adapted all sections that had 5rem to have 7rem when it's 1600 🤔 But I will let the designers take a look